new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -803,19 +838,19 @@ public static String createMiiImageAndVerify(String miiImageNameBase,
    * @return image name with tag
    */
   public static String createImageAndVerify(String imageNameBase,
-                                             String wdtModelFile,
-                                             String appName,
-                                             String modelPropFile,
-                                             String altModelDir,
-                                             String domainHome) {
+                                            String wdtModelFile,
+                                            String appName,
+                                            String modelPropFile,
+                                            String altModelDir,
+                                            String domainHome) {
 
     final List<String> wdtModelList = Collections.singletonList(MODEL_DIR + "/" + wdtModelFile);
     final List<String> appSrcDirList = Collections.singletonList(appName);
     final List<String> modelPropList = Collections.singletonList(altModelDir + "/" + modelPropFile);
 
     return createImageAndVerify(
-        imageNameBase, wdtModelList, appSrcDirList, modelPropList, WLS_BASE_IMAGE_NAME,
-        WLS_BASE_IMAGE_TAG, WLS, false, domainHome, false);
+            imageNameBase, wdtModelList, appSrcDirList, modelPropList, WLS_BASE_IMAGE_NAME,
+            WLS_BASE_IMAGE_TAG, WLS, false, domainHome, false);
```

Comment:
Are there tabs? I see it in a lot of files actually.

Reply:
done